commandline/WSL/install_guide.md

```diff
@@ -51,9 +51,9 @@ Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-L
 ```
 
 ### After enabling Windows Subsystem for Linux
-**Reboot when prompted**
+**Restart when prompted**

```

Comment:
Hey Susan, thanks for your suggested edits - could I ask why the preference for restart over reboot? We tend to refer to "restarting" Bash by closing and re-opening the Bash console window. But when we say reboot, we mean rebooting your whole PC.